font-types/src/fixed.rs

```diff
@@ -122,6 +135,7 @@ macro_rules! fixed_mul_div {
     ($ty:ty) => {
         impl $ty {
             /// Multiplies `self` by `a` and divides the product by `b`.
+            #[inline]
```

Comment:
not always?

Reply:
if intentional please comment why

Reply:
This one is large and not used that often, so I thought it best to leave it to compiler discretion